test_util/marathon.py

```diff
@@ -237,7 +237,7 @@ def _wait_for_pod_deployment(pod_id):
             r = self.get('v2/deployments')
             data = r.json()
             if len(data) > 0:
-                logging.info('Waiting for pod to be deployed %s', repr(data))
+                logging.info('Waiting for pod to be deployed %r', data)
```

Comment:
I have so many opinions here, one of which is: the actual grammar and content of that sentence is probably more important than which formatting syntax to use :-) But hey :ship: and :video_game: